src/Azure.Functions.Cli/Helpers/EolMessages.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Globalization;
+
+namespace Azure.Functions.Cli.Helpers
+{
+    internal class EolMessages
+    {
+        public static string GetEarlyEolCreateMessageForDotNet(string stackVersion, DateTime eol, string link = "")
```

Comment:
Shouldn't this be made more scalable so that you can reuse the same message for different runtimes (e.g. Python)?

Reply:
User can only select dotnet isolated version in core tools, others are always latest.